oap-server/server-starter/src/main/resources/ui-initialized-templates/elasticsearch/elasticsearch-index.json

```diff
@@ -0,0 +1,674 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+[
+  {
+    "id": "Elasticsearch-Index",
+    "configuration": {
+      "children":[
+        {
+          "x":0,
+          "y":6,
+          "w":12,
+          "h":13,
+          "i":"0",
+          "type":"Widget",
+          "id":"0",
+          "metricTypes":[
+            "readMetricsValues",
+            "readMetricsValues"
+          ],
+          "metrics":[
+            "meter_elasticsearch_index_stats_indexing_index_total_req_rate",
+            "meter_elasticsearch_index_stats_indexing_index_total_proc_rate"
+          ],
+          "moved":false,
+          "graph":{
+            "type":"Line",
+            "step":false,
+            "smooth":false,
+            "showSymbol":false,
+            "showXAxis":true,
+            "showYAxis":true
+          },
+          "widget":{
+            "title":"Indexing rate"
+          }
+        },
+        {
+          "x":12,
+          "y":6,
+          "w":12,
+          "h":13,
+          "i":"1",
+          "type":"Widget",
+          "id":"1",
+          "metricTypes":[
+            "readMetricsValues",
+            "readMetricsValues"
+          ],
+          "metrics":[
+            "meter_elasticsearch_index_stats_search_query_total_req_rate",
+            "meter_elasticsearch_index_stats_search_query_total_proc_rate"
+          ],
+          "moved":false,
+          "graph":{
+            "type":"Line",
+            "step":false,
+            "smooth":false,
+            "showSymbol":false,
```

Comment:
Notice:
The new UI submodule version will default set the `Line graph ` style to show the point on the line by setting `showSymbol:true`.
we need to set this config to `true` if the UI template is exported from an old version to keep the UI consistent.
please refer to: https://github.com/apache/skywalking/pull/10687

Reply:
If I re-save the data using the new UI (latest commit), will it take effect?

Reply:
I am using this repo (https://github.com/apache/skywalking-booster-ui) directl and did not use the submodule webapp.

Reply:
> If I re-save the data using the new UI (latest commit), will it take effect?

I think so :) @Fine0830

Reply:
> > If I re-save the data using the new UI (latest commit), will it take effect?
> 
> I think so :) @Fine0830

Sorry, re-save need you clean and re-create the dashboard, the better way manually changes it on the templates

Reply:
I can extract the UI template data from Elasticsearch after re-save,  copy it to the JSON file, so that I can manually modify it to avoid any possible mistakes. Because I am not very clear about how to write this json file.

Reply:
- Update the UI submodule to the latest.
- Find and replace `"showSymbol":false` to `"showSymbol":true` in your UI templates. 
- Clean the UI templates in your storage and restart OAP

then you can check if it's working.

Reply:
I don't think it will work. You would see this attribute is not changed at all. The only three options are 
1. Re-config all things from a clean env using the new UI.
2. Manually change configurations one by one using the new UI.
3. Change the field value from `false` to `true` manually.